scripts/clean-db-dump.sql

```diff
@@ -112,6 +112,51 @@ WHERE declaration IS NOT NULL
   AND declaration::varchar != 'null'
   AND declaration::varchar != '{}';
 
+-- Create some fake draft and closed brief responses on closed briefs using eligible suppliers
+-- Ensuring essential requirements evidence length matches that of the brief essential requirements
+INSERT INTO brief_responses (
+      data,
+      brief_id,
+      supplier_id,
+      created_at,
+      submitted_at
+)
+SELECT
+    -- Build a string that we convert to a json object for the 'data' column with varying numbers of evidence
+    (
+      '{' ||
+        '"essentialRequirements": [' ||
+          rtrim(repeat('{"evidence": "Some essential evidence."}, ', essential_requirements_len), ', ') ||
+        '], ' ||
+        '"niceToHaveRequirements": [' ||
+          rtrim(repeat('{"yesNo": "true", "evidence": "Some nice to have evidence."}, ', MOD(eligible_brief_supplier_pairings.supplier_id, 6)), ', ') ||
```

Comment:
So as we are doing MOD 6 on the supplier ID and MOD 5 on the nice to haves am I right in thinking we will have the following cases:
A response which is not submitted which has `niceToHaveRequirements`: []
A response which has been submitted which has `niceToHaveRequirements`: []
A response which is not submitted which has `niceToHaveRequirements`: [correct number of nice to haves]
A response which has been submitted which has `niceToHaveRequirements`: [correct number of nice to haves]

I don't think the first two responses are realistic data that we would see. If they had started an application but not yet answered the nice to have requirements question then we would see no key for nice to have requirements. If they had submitted the application, then they would have answered nice to have requirements and would have data in the list (even if it just said that "yesNo": false). 

You could go to the effort of trying to change the first two cases so you the key only exists with the correct number of nice to haves.
You could also simplify it so that they always have the key full of nice to haves.

Do correct me if any of my interpreting of your SQL is wrong. Also please do double check any of my assumptions if something you decide to change relies on them as this is all of the top of my head.